### DISCUSSION TITLE
Request for Assistance with Uploading Files in Supabase

### DISCUSSION THREAD

**Discussion by FamMap** [supabase/supabase#29913] [Questions] [answered]
I am experiencing an issue with file uploads in my project, "Fam Map," where I am using Supabase to store images. Unfortunately, the "Upload Files" functionality is not working as expected. When I attempt to upload files, a pop-up message appears stating that I need additional permissions to complete the upload process.

Could anyone 
please assist me in resolving this issue? I want to ensure that my project has the correct permissions set up so that I can successfully upload files to Supabase storage.

I would appreciate any guidance or support you can provide to help resolve this as soon as possible.
![Screenshot_1](https://github.com/user-attachments/assets/f3e0ad02-bcee-46d6-8f9a-2e24bd6ed094)

**Comment by silentworks** [comment]
Are you the project owner or were you added to the project as a Developer?

**Reply by FamMap** [reply]
I am the project owner.

**Reply by ryansburgoyne** [reply]
Is there a way to add permission for a developer to upload?

**Reply by silentworks** [reply]
@FamMap you should try restarting the project from this page https://supabase.com/dashboard/project/_/settings/general and if that doesn't help then try and contact Supabase support https://supabase.com/dashboard/support/new

**Reply by GaryAustin1** [reply]
I think this was a bug...
https://github.com/supabase/supabase/pull/29937

**Reply by FamMap** [reply, marked as answer]
I added authentication inside **_Storage policies_** under the _**Other policies under storage.objects**_. Now it works for all (owner, devloper, admin)
